rpm/pacemaker.spec.in

```diff
@@ -905,6 +905,7 @@ exit 0
 %{_datadir}/pacemaker/*.rng
 %{_datadir}/pacemaker/*.xsl
 %{_datadir}/pacemaker/api
+%{_datadir}/pacemaker/base
```

Comment:
I'm thinking this should be in the top schema directory rather than create a new subdirectory for just this